helpers/filterSeats.js

```diff
@@ -58,12 +59,32 @@ module.exports = function(dust) {
       });
     }
 
+    // use Transformer show config to add Gold, Silver or Bronze to packageRate
+    function assignColoursToBands(packageRate) {
+      // assign gold, silver, bronze to packages depending on their current priceBand
+      _.forEach(transformer, function(sectionValue, sectionKey) {
```

Comment:
`transformer` is maybe not the correct variable name, maybe `bandColours`, as you are only using the result from Transformer